### PR TITLE
Add readme for System.Linq

### PR DESCRIPTION
**Pull request by krwq**
cc: @eiriktsarpalis

### CODE REVIEW

File: src/libraries/System.Linq/README.md
Comment (eiriktsarpalis) [inline src/libraries/System.Linq/README.md:5]: I would probably also add that we're accepting analyzers, see https://github.com/dotnet/runtime/tree/main/src/libraries/System.Collections#contribution-bar

File: src/libraries/System.Linq/README.md
Comment (eiriktsarpalis) [inline src/libraries/System.Linq/README.md:2]: A few words about what is included in the library would be nice, even though most people are familiar with what Linq represents.

File: src/libraries/System.Linq/README.md
Comment (eiriktsarpalis) [inline src/libraries/System.Linq/README.md:10]: Do we need to link to the NuGet package? Last shipped package was back in 2016.

**Review by eiriktsarpalis — COMMENTED** [review]
Could you also add a README file to `System.Linq.Queryable`? It's part of the same area.

File: src/libraries/System.Linq/README.md
Comment (krwq) [reply src/libraries/System.Linq/README.md:5]:
```suggestion
- [x] [We consider new features, new APIs and performance changes](../../libraries/README.md#primary-bar)
- [x] [We consider PRs that target this library for new source code analyzers](../../libraries/README.md#secondary-bars)
- [x] [We don't accept refactoring changes due to new language features](../../libraries/README.md#secondary-bars)
```

Should we match this then? Apply suggestion if looks good

Comment (eiriktsarpalis) [reply src/libraries/System.Linq/README.md:5]: I'd probably remove the entry on new language features, since this library is closely tied to the language itself.

File: src/libraries/System.Linq/README.md
Comment (krwq) [reply src/libraries/System.Linq/README.md:2]: honestly I copy & pasted this from System.Reflection.Metadata, should we do this in some files and not in others. Is this doc even useful without a single sample?

Comment (eiriktsarpalis) [reply src/libraries/System.Linq/README.md:2]: I would try to include a short description of what the project contains rather than attempt to document it. In this case it would be something along the lines of "defines Linq extension methods for IEnumerable", vs. System.Linq.Queryable containing the equivalent extension methods for IQueryable.

File: src/libraries/System.Linq.Parallel/README.md
Comment (eiriktsarpalis) [inline src/libraries/System.Linq.Parallel/README.md:1]: We don't actually own area-System.Linq.Parallel, but unless the @dotnet/area-system-linq-parallel owners object we could add this on their behalf.

File: src/libraries/System.Linq.Parallel/README.md
Comment (eiriktsarpalis) [inline src/libraries/System.Linq.Parallel/README.md:9]: I'm not sure if PLINQ accepts new features anymore. cc @stephentoub

File: src/libraries/System.Linq.Queryable/README.md
Comment (eiriktsarpalis) [inline src/libraries/System.Linq.Queryable/README.md:5]:
```suggestion
Provides a set of extension methods for querying data structures that implement `IQueryable<T>`.
```

File: src/libraries/System.Linq/README.md
Comment (eiriktsarpalis) [inline src/libraries/System.Linq/README.md:5]: Nit: move description paragraph above the link to documentation.

File: src/libraries/System.Linq.Parallel/README.md
Comment (eiriktsarpalis) [inline src/libraries/System.Linq.Parallel/README.md:5]: Nit: description should be above link to documentation.

File: src/libraries/System.Linq.Queryable/README.md
Comment (eiriktsarpalis) [reply src/libraries/System.Linq.Queryable/README.md:5]: As with the other docs, should be moved above link to docs.

**Review by eiriktsarpalis — APPROVED** [review]
LGTM modulo minor feedback. Should seek approval from System.Linq.Parallel owners before merging.

File: src/libraries/System.Linq.Parallel/README.md
Comment (layomia) [reply src/libraries/System.Linq.Parallel/README.md:9]: Fwiw Git history doesn't show new features in over 2 years so I'd say maintenance mode.

File: src/libraries/System.Linq.Parallel/README.md
Comment (adamsitnik) [reply src/libraries/System.Linq.Parallel/README.md:1]: > We don't actually own area-System.Linq.Parallel, but unless the https://github.com/orgs/dotnet/teams/area-system-linq-parallel owners object we could add this on their behalf.

@eiriktsarpalis We don't object, thank you for your help!

File: src/libraries/System.Linq.Parallel/README.md
Comment (krwq) [reply src/libraries/System.Linq.Parallel/README.md:9]: Seems you're right changing to: `- [x] [We only consider lower-risk or high-impact fixes to maintain or improve quality](../../libraries/README.md#primary-bar)`

**Pull request merged**
